src/main/java/de/tum/in/www1/artemis/domain/iris/message/IrisMessageContent.java

```diff
@@ -0,0 +1,41 @@
+package de.tum.in.www1.artemis.domain.iris.message;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import de.tum.in.www1.artemis.domain.DomainObject;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import javax.annotation.Nullable;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "iris_message_content")
+@Inheritance(strategy = InheritanceType.JOINED)
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public abstract class IrisMessageContent extends DomainObject {
+    
+    @ManyToOne
+    @JsonIgnore
+    IrisMessage message;
```

Comment:
Please make this either protected or private.

Reply:
Package private is stricter than protected. If someone wants to subclass IrisMessageContent outside of this package then they can change it to protected as needed. Or is this an Artemis convention thing that I don't know about?